ModuleSystem/module_scene_props.py

```diff
@@ -2535,6 +2535,62 @@
 	( "boat_sail_on"                               ,0,"boat_sail_on","0",[]),
 	( "boat_sail_off"                              ,0,"boat_sail_off","0",[]),
 
+# from '': begin (OpenBRF)
+	( "tree_mirkwood_a"                            ,0,"tree_mirkwood_a","bo_tree_mallorn_a",[]),
+	( "tree_mirkwood_b"                            ,0,"tree_mirkwood_b","bo_tree_mallorn_b",[]),
+	( "tree_mirkwood_c"                            ,0,"tree_mirkwood_c","bo_tree_mallorn_c",[]),
+	( "tree_mirkwood_roots_1"                      ,0,"tree_mirkwood_roots_1","0",[]),
+	( "tree_mirkwood_roots_2"                      ,0,"tree_mirkwood_roots_2","0",[]),
+	( "tree_mirkwood_roots_3"                      ,0,"tree_mirkwood_roots_3","0",[]),
+	( "tree_mirkwood_roots_4"                      ,0,"tree_mirkwood_roots_4","0",[]),

```

Comment:
@In-Vain This OpenBRF formatting is a bit awkward, but will do. Try to keep the simple prop per line to make it more readable. Complex things with triggers like the one below are fine, of course.